extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -628,7 +628,7 @@ function axWFactorySaveVariable() {
 					// apply changes to all wikis with given tag
 					$tagsQuery = new WikiFactoryTagsQuery( $tag_name );
 					foreach ( $tagsQuery->doQuery() as $tagged_wiki_id ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)